### PR TITLE
python311Packages.pygls: 1.1.1 -> 1.1.2

### PR DESCRIPTION
**Pull request by kira-bruneau**
## Description of changes

https://github.com/openlawlibrary/pygls/blob/v1.1.2/CHANGELOG.md#111---2023-10-06

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by kira-bruneau** [comment]
Result of `nixpkgs-review pr 265717` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>17 packages built:</summary>
  <ul>
    <li>bitbake-language-server</li>
    <li>bitbake-language-server.dist</li>
    <li>cmake-language-server</li>
    <li>cmake-language-server.dist</li>
    <li>python310Packages.jedi-language-server</li>
    <li>python310Packages.jedi-language-server.dist</li>
    <li>python310Packages.pygls</li>
    <li>python310Packages.pygls.dist</li>
    <li>python311Packages.jedi-language-server</li>
    <li>python311Packages.jedi-language-server.dist</li>
    <li>python311Packages.pygls</li>
    <li>python311Packages.pygls.dist</li>
    <li>ruff-lsp</li>
    <li>ruff-lsp.dist</li>
    <li>vimPlugins.YouCompleteMe</li>
    <li>vscode-extensions.ms-python.python</li>
    <li>ycmd</li>
  </ul>
</details>

File: pkgs/development/python-modules/pygls/default.nix [hunk 1]
```diff
@@ -49,9 +49,9 @@ buildPythonPackage rec {
   pythonImportsCheck = [ "pygls" ];
 
   meta = with lib; {
-    changelog = "https://github.com/openlawlibrary/pygls/blob/${src.rev}/CHANGELOG.md";
     description = "Pythonic generic implementation of the Language Server Protocol";
     homepage = "https://github.com/openlawlibrary/pygls";
+    changelog = "https://github.com/openlawlibrary/pygls/blob/v${src.rev}/CHANGELOG.md";
```
Comment (natsukium) [inline pkgs/development/python-modules/pygls/default.nix:54]:
```suggestion
    changelog = "https://github.com/openlawlibrary/pygls/blob/${src.rev}/CHANGELOG.md";
```
`src.rev` already includes the prefix `v`.

Comment (kira-bruneau) [reply pkgs/development/python-modules/pygls/default.nix]: oops, thanks for catching that!!

**Review by natsukium — APPROVED** [review]
LGTM

**Pull request merged**